tests/integration/s3-offload/src/test/java/org/apache/pulsar/tests/integration/TestS3Offload.java

```diff
@@ -179,28 +180,20 @@ public void testPublishOffloadAndConsumeViaCLI() throws Exception {
         }
     }
 
-    @Test
-    public void testPublishOffloadAndConsumeViaThreshold() throws Exception {
+    @Test(dataProvider =  "ServiceAndAdminUrls")
+    public void testPublishOffloadAndConsumeViaThreshold(String serviceUrl, String adminUrl) throws Exception {
```

Comment:
who is using adminUrl

Reply:
https://github.com/apache/incubator-pulsar/pull/2101/files/77f840237aa80fef55477c16f038ef5ef6ec608a#diff-79f7bfa7dd1879a88347464ef101a81dR138